packages/ui/components/ui/tx/view/output.tsx

```diff
@@ -1,22 +1,22 @@
 import { OutputView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/shielded_pool/v1/shielded_pool_pb';
 import { ViewBox } from './viewbox';
 import { ValueViewComponent } from './value';
-import { AddressViewComponent } from './address-view';
+import { ValueWithAddress } from './value-with-address';
+import { getNote } from '@penumbra-zone/getters/src/output-view';
+import { getAddress } from '@penumbra-zone/getters/src/note-view';
 
 export const OutputViewComponent = ({ value }: { value: OutputView }) => {
   if (value.outputView.case === 'visible') {
-    const note = value.outputView.value.note!;
+    const note = getNote(value);
+    const address = getAddress(note);
+
     return (
       <ViewBox
         label='Output'
         visibleContent={
-          <div className='flex flex-col justify-between gap-2 sm:flex-row sm:gap-0'>
+          <ValueWithAddress addressView={address} label='to'>
```

Comment:
Extracted this markup to `ValueWithAddress` — see the screenshot in my comment on the `ValueWithAddress` code below